README.md

```diff
@@ -1,3 +1,5 @@
+[![Puppet Forge](http://img.shields.io/puppetforge/v/katello/gutterball.svg)](https://forge.puppetlabs.com/katello/gutterball)
```

Comment:
Yea img.shields.io seems like its having trouble the last day or so.  Our extra ~10 badges broke it all I guess :smile: